src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -389,6 +392,10 @@ record = _serverStore.Cluster.ReadDatabase(context, Name, out index);
                 {
                     var lastCompletedClusterTransactionIndex = DocumentsStorage.ReadLastCompletedClusterTransactionIndex(ctx.Transaction.InnerTransaction);
                     ClusterWideTransactionIndexWaiter.SetAndNotifyListenersIfHigher(lastCompletedClusterTransactionIndex);
+
+                    var lastCounterFixed = DocumentsStorage.ReadLastFixedCounterKey(ctx.Transaction.InnerTransaction);
+                    if (lastCounterFixed != CountersRepairTask.Completed)
+                        _ = Task.Run(() => CountersRepairTask.Start(lastCounterFixed), DatabaseShutdown);
```

Comment:
pass a token to the `Start` itself, the token for the task is relevant only if the task wasn't scheduled yet

Reply:
done